R/model.R

```diff
@@ -1044,19 +940,16 @@ CmdStanModel$set("public", name = "sample", value = sample_method)
 #'   )
 #'   ```
 #'
-#' @section Arguments unique to the `sample_mpi` method:
-#'   * `mpi_cmd`: (character vector) The MPI launcher used for launching MPI processes.
-#'     The default launcher is `"mpiexec"`.
-#'   * `mpi_args`: (list) A list of arguments to use when launching MPI processes.
-#'     For example, `mpi_args = list("n" = 4)` launches the executable as
-#'     `mpiexec -n 4 model_executable`, followed by CmdStan arguments
-#'     for the model executable.
-#'
-#'    All other arguments are the same as for [`$sample()`][model-method-sample]
-#'    except `$sample_mpi()` does not have arguments `threads_per_chain` or
-#'    `parallel_chains`.
+#' @template model-common-args
+#' @template model-sample-args
```

Comment:
sample_mpi and sample share same args template

Reply:
This isn't needed anymore. We can use `@inheritParams` after the change I made to fix the usage sections. (see comments below)